README.md

```diff
@@ -940,6 +940,7 @@ _Related: [Groupware](#groupware)_
 
 **[`^        back to top        ^`](#)**
 
+- [0x0](https://0x0.st/) - No-bullshit file hosting and URL shortening service ([Demo](https://0x0.st/) [Source Code](https://git.0x0.st/mia/0x0)) `EUPL-1.2` `Python`
```

Comment:
Congratulations on your contribution!

One change to make it to remove the duplicate demo link and leave it as the main site.